package.json

```
@@ -1,6 +1,6 @@
 {
     "name": "@aboutyou/backbone",
-    "version": "13.10.3",
+    "version": "13.10.4",
```

```suggestion
    "version": "13.11.0",
```

It's a new feature, so it should be a minor release and not a patch

---

Changed

I just matched Changelog. Fixed changlog as well